vllm/engine/arg_utils.py

```diff
@@ -1,3 +1,4 @@
+# ruff: noqa: E501
```

Comment:
Over time this would make this file less and less readable

```suggestion
```

Reply:
I have removed this.